index.html

```diff
@@ -6,13 +6,191 @@
+          <img src="icons/event1.png" alt="Workshop icon"/>
+          <h4>DJ's Conference</h4>
+          <p>
+            Meeting with the top world DJs 
+            Is simply dummy text of the printing  
+            has been the 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event2.png" alt="Workshop icon"/>
+          <h4>Producer's Seminar</h4>
+          <p>
+            EDM Producers share their experience 
+            Is simply dummy text of the printing  
+            has been the 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event3.png" alt="Workshop icon"/>
+          <h4>Social Media</h4>
+          <p>
+            Media experts share their knowledge  
+            and typesetting industry. Lorem Ipsum 
+            has been 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event4.png" alt="Workshop icon"/>
+          <h4>Lounge set</h4>
+          <p>
+            Above & beyond launches ABGT 600 
+            and typesetting industry. Lorem Ipsum 
+            has been the
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event5.png" alt="Workshop icon"/>
+          <h4>Tiesto's case</h4>
+          <p>
+            How the world's most famous DJ made it 
+            Is simply dummy text of the printing 
+            has been 
+          </p>
+        </li>
+      </ul>
+      <div class="see-program">
+        <h4>SEE THE WHOLE PROGRAM</h4>
+      </div>
+      <button class="join" type="button">
+        Join ECM 2022 now
+      </button>
+    </section> <!-- END OF MAIN -->
+
+    <section class="talks" id="Talks"> <!-- START OF TALKS -->
+      <div class="h3-header">
+        <h3>EDM Professionals</h3>
+        <div class="dash">  </div>
+      </div>
+      <ul class="speakers" id="Speakers">
+        
+        <!-- HERE GO THE DYNAMIC CARDS -->
+
+      </ul>
+      <div class="more">
+        <button class="more-speakers" type="button">
+          MORE
+          <img src="icons/arrow-down.png" alt="">
+        </button>
+      </div>
+    </section> <!-- END OF TALKS -->
+
+    <section class="partners" id="Partners"> <!-- START OF PARTNERS -->
+      <div class="h3-header">
+        <h3>
+          Partners
+        </h3>
+        <div class="dash">  </div>
+      </div>
+      
+      <div class="logo-carousel">
+        <div class="logo-item">
+          <img src="../icons/partner1.png" alt=""/>
+        </div>
+        <div class="logo-item">
+          <img src="../icons/partner4.png" alt=""/>
+        </div>
+        <div class="logo-item">
+          <img src="../icons/partner3.png" alt=""/>
+        </div>
+        <div class="logo-item">
+          <img src="../icons/partner2.png" alt=""/>
+        </div>
+        <div class="logo-item">
+          <img src="../icons/partner5.png" alt=""/>
```

Comment:
- [ ] Kindly your `partners section` appears on the about page but doesn't on the home page. Just remove one `.` from the `src="../icons/partner2.png"` make it `src="./icons/partner2.png"`.

![image](https://user-images.githubusercontent.com/35535130/157727207-b5a38847-8896-4637-a5e1-2510b5c2c9ee.png)